music21/figuredBass/notation.py

```diff
@@ -412,14 +452,18 @@ class Figure(prebase.ProtoM21Object):
             ''',
     }
 
-    def __init__(self, number=1, modifierString=None):
+    def __init__(self, number=1, modifierString=None, isExtender=None):
         self.number = number
         self.modifierString = modifierString
         self.modifier = Modifier(modifierString)
+        if self.number == '_':
```

Comment:
This whole block could be replaced with:

`self.isExtender = (self.number == "_")`

Reply:
Right.